pythonforandroid/bootstraps/pygame/__init__.py

```diff
@@ -40,21 +43,14 @@ def run_distribute(self):
             info('Copying python distribution')
             hostpython = sh.Command(self.ctx.hostpython)
             # AND: This *doesn't* need to be in arm env?
-            shprint(hostpython, '-OO', '-m', 'compileall', self.ctx.get_python_install_dir())
+            shprint(hostpython, '-OO', '-m', 'compileall', self.ctx.get_python_install_dir(),
+                    _tail=10, _filterout="^Listing", _critical=True)
```

Comment:
Okay, ignore my last reply, I understand better now and it seems good (though is there a reason for picking this tail number in particular?).